morse.rb

```diff
@@ -0,0 +1,45 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(char)
+  MORSE_CODE[char]
+end
+
+def decode_word(word)
+  letters = word.chars
+  letters.map { |letter| decode_char(letter) }.join
+end
+
+def decode_message(message)
+  words = message.chars
+  words.map { |word| decode_word(word) }.join
+end
+
+puts decode_message('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```

Comment:
- [ ] We are getting wrong result for your `decode_message` method. Kindly fix this.
![Screenshot from 2023-04-07 16-08-48](https://user-images.githubusercontent.com/65860678/230624689-ef7b4ad4-3c82-4e28-8b03-1119fc543a5b.png)